src/Corrade/Containers/Test/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ corrade_add_test(ContainersStringViewTest StringViewTest.cpp LIBRARIES CorradeUt
 corrade_add_test(ContainersStringViewStlTest StringViewStlTest.cpp)
 corrade_add_test(ContainersTripleTest TripleTest.cpp)
 corrade_add_test(ContainersTripleStlTest TripleStlTest.cpp)
+corrade_add_test(ContainersTupleStlTest TupleStlTest.cpp)
```

Comment:
To make the tests on ancient compilers stop failung, move this inside the C++17 `if()` below, same place where you set the C++17 standard